tests/baselines/reference/compoundAdditionAssignmentWithInvalidOperands.errors.txt

```diff
@@ -38,13 +38,13 @@ tests/cases/conformance/expressions/assignmentOperator/compoundAdditionAssignmen
 !!! error TS2365: Operator '+=' cannot be applied to types 'boolean' and 'void'.
     x1 += true;
     ~~~~~~~~~~
-!!! error TS2365: Operator '+=' cannot be applied to types 'boolean' and 'true'.
+!!! error TS2365: Operator '+=' cannot be applied to types 'boolean' and 'boolean'.

```

Comment:
Ack - I just noticed this and this definitely isn't right.

Reply:
Yeah looks right to me